src/proc/y411-converter.h

```diff
@@ -7,7 +7,7 @@
 
 namespace librealsense
 {
-    class y411_converter : public functional_processing_block
+    class LRS_EXTENSION_API y411_converter : public functional_processing_block
     {
     public:
         y411_converter(rs2_format target_format)
```

Comment:
Target format isn't used and so only RGB8 is supported....
Maybe we shouldn't accept it as an argument?

Reply:
I think we need to add a task to add BGR , GRB ...